instrumentation/pekko/pekko-http-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkohttp/v1_0/server/route/RouteConcatenationInstrumentation.java

```diff
@@ -25,41 +27,17 @@ public ElementMatcher<TypeDescription> typeMatcher() {
   @Override
   public void transform(TypeTransformer transformer) {
     transformer.applyAdviceToMethod(
-        named("$anonfun$$tilde$1"), this.getClass().getName() + "$ApplyAdvice");
-    transformer.applyAdviceToMethod(
-        named("$anonfun$$tilde$2"), this.getClass().getName() + "$Apply2Advice");
+        MethodDescription::isConstructor, this.getClass().getName() + "$ApplyAdvice");
+    transformer.applyAdviceToMethod(named("$tilde"), this.getClass().getName() + "$ApplyAdvice");
   }
 
-  @SuppressWarnings("unused")
```

Comment:
usually we add `@SuppressWarnings("unused")` to advice classes as they are not directly used